Sources/Datadog/DDTracer.swift

```diff
@@ -52,6 +54,7 @@ public class DDTracer: Tracer {
 
     internal init(spanOutput: SpanOutput) {
         self.spanOutput = spanOutput
+        self.queue = DispatchQueue(label: "com.datadoghq.tracer", qos: .userInteractive)
```

Comment:
this way of creating queues is discouraged in [Apple docs](https://developer.apple.com/documentation/dispatch/dispatchqueue)
could we set the target to a global dispatch queue as suggested?

Reply:
👍 switched to:
```swift
self.queue = DispatchQueue(
    label: ...,
    target: .global(qos: .userInteractive)
)
```
both for `DDTracer` and `Logger`.